spec/Section 3 -- Type System.md

```diff
@@ -1755,7 +1756,12 @@ A GraphQL schema describes directives which are used to annotate various parts
 of a GraphQL document as an indicator that they should be evaluated differently
 by a validator, executor, or client tool such as a code generator.
 
-GraphQL implementations should provide the `@skip` and `@include` directives.
+GraphQL implementations should provide the `@skip` and `@include` directives. 
+
+GraphQL implementations are not required to implement the `@defer` and `@stream` 
```

Comment:
I assume it's legal for an implementation to implement `@defer` but not `@stream` (or vice-versa)? It's a bit ambiguous here whether they have to go together or not.

Reply:
I updated the wording here which I think should make it more clear that either or both can be implemented.